packages/core/src/Stores/segment-store.js

```diff
@@ -0,0 +1,67 @@
+import {
+    action,
+    computed }                  from 'mobx';
+import BinarySocket             from '_common/base/socket_base';
+import { isLoginPages }         from '_common/base/login';
+import { get as getLanguage }   from '_common/language';
+import BaseStore                from './base-store';
+import { getAppId }             from '../config';
+
+export default class SegmentStore extends BaseStore {
+    // only available on production
+    is_applicable = /^(16929)$/.test(getAppId());
```

Comment:
will this work for bot since the app id is 19111?

Reply:
yes, you're right, it has different app_id